hack/e2e.sh

```diff
@@ -141,13 +141,122 @@ function test_elasticsearchcluster() {
 
 test_elasticsearchcluster
 
+function cql_connect() {
+    local namespace="${1}"
+    local host="${2}"
+    local port="${3}"
+    # Attempt to negotiate a CQL connection.
+    # No queries are performed.
+    # stdin=false (the default) ensures that cqlsh does not go into interactive
+    # mode.
+    # XXX: This uses the standard Cassandra Docker image rather than the
+    # gcr.io/google-samples/cassandra image used in the Cassandra chart, becasue
+    # cqlsh is missing some dependencies in that image.
+    kubectl \
+        run \
+        "cql-responding-${RANDOM}" \
+        --namespace="${USER_NAMESPACE}" \
```

Comment:
USER_NAMESPACE here should be ${namespace}

Reply:
Ah, yeah, that's a mistake, but it doesn't affect the test results.
`USER_NAMESPACE` is redefined as a local in `test_cassandracluster`  and it'll be using that.
I propose I fix this in a followup branch rather than have to go through another cycle of E2E tests.